lib/preview-web/src/PreviewWeb.tsx

```diff
@@ -209,8 +209,10 @@ export class PreviewWeb<TFramework extends AnyFramework> extends Preview<TFramew
     if (this.currentRender instanceof DocsRender) await this.currentRender.rerender();
   }
 
-  async onPreloadStories(ids: string[]) {
-    await Promise.all(ids.map((id) => this.storyStore.loadStory({ storyId: id })));
+  async onPreloadStories({ ids }: { ids: string[] }) {
+    await Promise.all(
+      ids.map((id) => this.storyStore.loadStory({ storyId: id }).catch(() => undefined))
```

Comment:
Is this still needed? Does the targetting work?

Reply:
@tmeasday so it's possible that we're trying to preload a story in a ref we haven't loaded the iframe for yet.

Because of the way the targeting works, if we can't find the targeted iframe, we'll use the currently active iframe.
Which can cause the event to be targeted to the wrong iframe, causing an error if the storyId does not exists there.

Therefore I opted to catch the error thrown.
(and try to target the correct iframe)

Reply:
> I did notice the target attribute doesn't seem consistent:

great catch, I think I fixed that

Reply:
OK, seems reasonable, can you make a comment about that?

Reply:
Done!